lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -2152,8 +2152,8 @@ namespace Js
                 {
                     Assert((flags & Data) == Data && (flags & Writable) == None);
 
-                    JavascriptError::ThrowCantAssignIfStrictMode(propertyOperationFlags, requestContext);
                     requestContext->GetThreadContext()->AddImplicitCallFlags(ImplicitCall_NoOpSet);
+                    JavascriptError::ThrowCantAssignIfStrictMode(propertyOperationFlags, requestContext);
```

Comment:
It is a little confusing how this case should be handled.
The implicit call possibly already happened and we want to throw/bailout.
With my change, in this scenario it means that if we disabled implicit calls we won't throw and bailout instead.
The question is, should we prioritize bailing out or throwing ?